scripts/coursera_quiz_conversion.R

```diff
@@ -0,0 +1,79 @@
+# Need here and R.utils
+# For testing use tx_i  <- readLines(here("quizzes", "quiz_ch1.md"))
+
+# This is functionalized but running like script for now till we decide where to put this:
+# run by  pasting `Rscript scripts/coursera_quiz_conversion.R` in the terminal
+
+library(here)
+library(R.utils)
+
+
+coursera_quizzes <- function(path= here("quizzes"), # we might want to update to manuscript
+         verbose = TRUE) {
+  leanpub_quizzes = list.files(
+    pattern = (".md"),
+    ignore.case = TRUE,
+    path = path,
+    full.names = FALSE)
+  if (length(leanpub_quizzes) < 1) {
+    warning("You need quiz files in your specified path dir")
+  }
+  print(leanpub_quizzes)
+  
+  for(i in leanpub_quizzes){
+    tx_i  <- readLines(paste0(path,"/", i))
```

Comment:
```suggestion
    tx_i  <- readLines(file.path(path, a_quiz))
```
If you use `file.path()` then this script will be able to be run in windows or mac or whatever.

Reply:
Sounds good, thanks!